modules/subscriptions/subscriptions.css

```diff
@@ -1,7 +1,47 @@
+.subscription-form {
+	margin: 0 0 0;
+	padding: 20px;
+	border: 4px solid black;
+	background: rgba( 255, 255, 255, .9 );
+}
+
+.subscription-form__title {
+	margin: 0;
+}
+
+.subscription-form__title.subscription-form__title input,
+.subscription-form__text.subscription-form__text textarea  {
+	width: 100%;
+	margin: 0 0 0;
+	outline: 0;
+	box-shadow: none;
+	margin-bottom: 16px;
+}
+
+.subscription-form__title.subscription-form__title input {
+	font-size: 20px;
+	font-weight: 600;
+}
+
+.subscription-form__text p {
+	font-size: 14px;
+}
+
+p.subscription-form__subscribers {
+	font-size: 12px;
+	color: #555d66;
+}
+
+.subscription-form__email-label {
+	font-size: 14px;
+	font-weight: 600;
+	margin-left: 2px;
+}
+
 #subscribe-email input {
-	width: 95%;
+	width: 100%;
 }
 
-.comment-subscription-form .subscribe-label {
```

Comment:
Could these changes negatively affect existing forms with custom styles?

Reply:
The short answer is, I don't know. Are they even effecting the front end styles? I didn't think they were.

Reply:
I believe so -- that stylesheet had previously been enqueued with the widget on the front end iirc.